cli/azd/pkg/oneauth/bridge/vcpkg-configuration.json

```diff
@@ -0,0 +1,17 @@
+{
+  "default-registry": {
+    "kind": "git",
+    "repository": "https://github.com/Microsoft/vcpkg",
+    "baseline": "c6c59e33f39b6c5588fb5884537d31c51dd947a9"
+  },
+  "registries": [
+    {
+      "kind": "git",
+      "repository": "https://office.visualstudio.com/OneAuth/_git/vcpkg-registry",
```

Comment:
What (if anything) do we need developers to do to access this repository?  I assume it will only be accessible to internal MSFT folks.

Reply:
Yes, it's internal only. If memory serves, I already had access, I suppose by virtue of membership in some other org. I'm sure we can get access for anyone on the team who needs it.